tests/unit/src/test/resources/documentSymbol/example/AnonymousClasses.scala

```diff
@@ -0,0 +1,3 @@
+/*example*/package example
+
+/*AnonymousClasses*/class AnonymousClasses {}
```

Comment:
No ranges, yet.

@olafurpg The issue I have is that I don't have meaningful end positions in the `Semanticdbs.printTextDocument` method.

When I convert from `DocumentSymbol` to `SymbolOccurrence` I discard the end positions in order to make `printTextDocument` work.

Ideally, I would like to print the true end position of a document symbol. It's a meaningful test because ranges are used by editors to determine whether the cursor is "inside" a symbol and highlight the corresponding outline entry

Reply:
What if the symbol text inside the comment has the format `$KIND:$END_LINE`?

```scala
/*Namespace:20*/package example
 /*Class:10*/class AnonymousClasses {} 
```

Reply:
Same issue, the end line is equal to the start line in the `SymbolOccurrence` (or it would mess up with offset computation in `printTextDocument`)

Reply:
Can you elaborate? The end offset for `SymbolOccurrence.range` is not important, it can be the same as the start offset.

Reply:
Suppose I have a `DocumentSymbol` which spans until the end of the file (e.g. the symbol for a class definition).

When converting this symbol to a `SymbolOccurence`, I have to make the end position (line/char) identical to the end position, or the offset calculation will go wrong. Specifically, the issue is with these lines:

https://github.com/scalameta/metals/blob/d4199aa5f5a714e0b748e8e7e019e3049b97f21d/mtags/src/main/scala/scala/meta/internal/mtags/Semanticdbs.scala#L87-L94

If I pass in a `SymbolOccurrence` using the "correct" end position, the `.substring` call goes out of range.

Reply:
I left a comment for how to print the end line in `SymbolOccurrence.symbol`